### PR TITLE
enhance POD layout when printing

### PR DESCRIPTION
**Pull request by ThisUsedToBeAnEmail**

### CODE REVIEW

File: root/wrapper.html [hunk 1]
```diff
@@ -163,7 +163,7 @@
             </div>
         </nav>
 
-        <div class="container-fluid">
+        <div class="container-fluid" id="page-<% c.action.namespace %>">
```
Comment (haarg) [inline root/wrapper.html:166]: We try to avoid unneeded ids so they don't conflict with pod content.

File: root/static/less/print.less [hunk 2]
```diff
@@ -1,4 +1,44 @@
...
     .pod { margin: 0px; border: 0px solid #000; padding: 0px}
     .home, .header, .search-bar, .box-right, .footer, a.favorite {display:none}
+    #page-pod {
+        ul.nav-list.slidepanel.sticky-panel-top,
+        ul.nav-list.slidepanel.sticky-panel-bottom.sticky-panel-sticky,
+        ul.nav-list.box-right.hidden-phone.dependencies,
+        form.search-form.form-horizontal,
+        .head-small-logo.col-md-3,
+        span.dropdown,
+        div#libwww-perl-fav,
```
Comment (haarg) [inline root/static/less/print.less:11]: Some of these selectors seem a bit weird.

**Comment by haarg** [comment]
Addresses #1786

File: root/wrapper.html (hunk 1 above)
Comment (ThisUsedToBeAnEmail) [reply root/wrapper.html:166]: This allows for page specific CSS without including custom stylesheets.

Comment (ThisUsedToBeAnEmail) [reply root/wrapper.html:166]: Perhaps I shouldn't be using c.action.namespace to achieve this.

Comment (haarg) [reply root/wrapper.html:166]: classes should be preferred to ids.

`c.action.namespace` is probably the wrong thing.  I'd say anything accessing `c` is probably wrong. Not sure what the best option is though. Possibly something injected by the view class.

File: root/static/less/print.less [hunk 3]
```diff
@@ -1,4 +1,45 @@
...
     .pod { margin: 0px; border: 0px solid #000; padding: 0px}
     .home, .header, .search-bar, .box-right, .footer, a.favorite {display:none}
+    #page-pod {
+        .sticky-panel-top,
+        .sticky-panel-sticky,
+        .dependencies,
+        .search-form,
+        .head-small-logo,
+        .dropdown,
+        #libwww-perl-fav,
```
Comment (haarg) [inline root/static/less/print.less:11]: This selector still doesn't make any sense.

File: root/wrapper.html (hunk 1 above)
Comment (ThisUsedToBeAnEmail) [reply root/wrapper.html]: I've changed the ID to be a class, however I'm unsure whether you will like the new approach.

File: root/static/less/print.less (hunk 3 above)
Comment (ThisUsedToBeAnEmail) [reply root/static/less/print.less]: My bad, I've changed that ID to a class .logged_in

**Comment by oalders** [comment]
Thanks @ThisUsedToBeAnEmail! I've merged the commits at the command line.

**Pull request closed without merging**